wurst/systems/spells/gatherer/Radars.wurst

```diff
@@ -0,0 +1,339 @@
+package Radars
+
+import AbilityObjEditing
+import Assets
+import ClosureEvents
+import ChatCommands
+import ClosureForGroups
+import GameConfig
+import ClosuresForItems
+import ClosureTimers
+
+@configurable constant string SPELL_ICON = Icons.bTNSpy
+@configurable constant real COOLDOWN = 60
+@configurable constant real CASTTIME = 1
+@configurable constant real RADAR_RANGE_BASE = 3300
+@configurable constant real RADAR_RANGE_PER_LEVEL = 200
+@configurable constant real MAX_RANGE = 5000
+@configurable constant real REVEAL_DURATION = 10
+@configurable constant int MANACOST = 15
+@configurable constant string array SPELLBOOKSPELLS = ["XGFT, XGFZ, XGFR, XGFC", "XGFT, XGFZ, XGFR, XGFC, XGFF, XGFS", "XGFT, XGFZ, XGFR, XGFC, XGFF, XGFS, XGFM"]
+@configurable constant int ABILITY_ID_FINDTINDER = 'XGFT'
+@configurable constant int ABILITY_ID_FINDSTICK = 'XGFZ'
+@configurable constant int ABILITY_ID_FINDMUSHROOM = 'XGFR'
+@configurable constant int ABILITY_ID_FINDCLAYBALL = 'XGFC'
+@configurable constant int ABILITY_ID_FINDFLINT = 'XGFF' 
+@configurable constant int ABILITY_ID_FINDSTONE = 'XGFS'
+@configurable constant int ABILITY_ID_FINDMANA = 'XGFM'
+@configurable constant int ABILITY_ID_SPELLBOOK = 'XGRB'
+@configurable constant string TOOLTIP_NORM_FINDTINDER = "|c00ffcc00Q|r - Find Tinder"
+@configurable constant string TOOLTIP_EXT_FINDTINDER = "Pings all nearby tinder in the minimap and reveals them for a short duration."
+@configurable constant string TOOLTIP_NORM_FINDSTICK = "|c00ffcc00W|r - Find Stick"
+@configurable constant string TOOLTIP_EXT_FINDSTICK = "Pings all nearby sticks in the minimap and reveals them for a short duration."
+@configurable constant string TOOLTIP_NORM_FINDMUSHROOM = "|c00ffcc00E|r - Find Mushroom"
+@configurable constant string TOOLTIP_EXT_FINDMUSHROOM = "Pings all nearby mushrooms in the minimap and reveals them for a short duration."
+@configurable constant string TOOLTIP_NORM_FINDCLAYBALL = "|c00ffcc00R|r - Find Clay Ball"
+@configurable constant string TOOLTIP_EXT_FINDCLAYBALL = "Pings all nearby clay balls in the minimap and reveals them for a short duration."
+@configurable constant string TOOLTIP_NORM_FINDFLINT = "|c00ffcc00A|r - Find Flint"
+@configurable constant string TOOLTIP_EXT_FINDFLINT = "Pings all nearby flints in the minimap and reveals them for a short duration."
+@configurable constant string TOOLTIP_NORM_FINDSTONE = "|c00ffcc00S|r - Find Stone"
+@configurable constant string TOOLTIP_EXT_FINDSTONE = "Pings all nearby stones in the minimap and reveals them for a short duration."
+@configurable constant string TOOLTIP_NORM_FINDMANA = "|c00ffcc00D|r - Find Mana Crystals"
+@configurable constant string TOOLTIP_EXT_FINDMANA = "Pings all nearby mana crystals in the minimap and reveals them for a short duration."
```

Comment:
`@configurable` is only necessary if we put this code into a library. For local code we don't need to add it. Also, instead of types, lets use `let` as per the Wurst conventions: https://wurstlang.org/manual.html#idiomatic-use-of-language-features